### PR TITLE
Initial feature set

### PR DESCRIPTION
**Pull request by Fcmam5**
### Change:

- Added contributing guides and updated README 
- Exposed a first "example" API method.

### Checklist:

- [x] Fetch the dataset and add a script to update it each time
- [x] Add very basic features to get this project started
- [x] Document the added features

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+const getWilayaList = require('./api/getWilayaList');
+
+const data = require('../data/WilayaList.json');
+
+const _getData = () => ([...data]);
+
+
+module.exports = {
+  getWilayaList: getWilayaList(_getData())
```
Comment (Fcmam5) [inline src/index.js:9]: I went for this solution of passing the dataset-clone to the internal functions to make testing easier. Calling this clone function will prevent us from having weird results. 

We can discuss whether to keep it like this or force cloning before usage inside the functions (for sure we should ensure mutating this parameter anyway)

File: .eslintrc.json [hunk 2]
```diff
@@ -23,14 +23,15 @@
         "ecmaVersion": 2018
     },
     "rules": {
+        "semi": "error",
```
Comment (ZibanPirate) [inline .eslintrc.json]: indentation space here is 4, although in prettierrc configuration is set to 2 🧐

File: package.json [hunk 3]
```diff
@@ -1,12 +1,14 @@
 {
-  "name": "leblad-sdk",
```
Comment (ZibanPirate) [inline package.json:2]: usually npm packages names are simple, plus currently the name `leblad` is available, i think it's better to keep it `leblad` what u think?

File: src/index.js (hunk 1 above)
Comment (ZibanPirate) [reply src/index.js:9]: that's right, in this complex dataset mutation is a big problem, well thought 👌🏼

**Review by ZibanPirate — CHANGES_REQUESTED** [review]
please check the comments @Fcmam5 , also, can you please add 
```ini
indent_size = 2
indent_style = space
```
to all files in `.editorconfig` so that all files follow the same style

hope you are not a `tab` guy 🙏

File: package.json (hunk 3 above)
Comment (Fcmam5) [reply package.json:2]: That's what I did 😅  see [L2](https://github.com/dzcode-io/leblad/pull/2/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R2) addition: 
```
+  "name": "leblad",
```

File: .eslintrc.json (hunk 2 above)
Comment (Fcmam5) [reply .eslintrc.json]: Oups 🙊

**Review by ZibanPirate — APPROVED** [review]

**Pull request merged**